dgraph/cmd/zero/zero.go

```diff
@@ -79,6 +79,7 @@ type Server struct {
 	blockCommitsOn *sync.Map
 
 	checkpointPerGroup map[uint32]uint64
+	pb.UnimplementedZeroServer
```

Comment:
probably the same issue here

Reply:
This is introduced in newer versions of protoc-gen-grpc-go. We need to embed the pb.UnimplementedZeroServer struct to ensure forward compatibility of the server. We can generate code without forward compatibility by setting the option --go-grpc_out=require_unimplemented_servers=false: on the compiler.